src/browser.ts

```diff
@@ -82,31 +90,49 @@ export const launchBrowser = async (
   // if we are on linux we spawn the browser directly and attach the on exit callback
   if (getPlatform() === 'linux') {
     const browserProc = spawn(path, args)
+    browserProc.on('error', sendBrowserLaunchFailedEvent)
     browserProc.once('exit', sendBrowserClosedEvent)
     return browserProc
   }
 
   // macOS & windows
-  return launch({
+  const browserProc = launch({
     executablePath: path,
     args: args,
     onExit: sendBrowserClosedEvent,
   })
+  browserProc.nodeProcess.on('error', sendBrowserLaunchFailedEvent)
```

Comment:
`launch` uses `spawn` under the hood, which handles errors asynchronously, so try/catch doesn't work here and we need to listen on `error` instead.